rspec-mode.el

```diff
@@ -505,7 +505,7 @@ to navigate to the example or method corresponding to point."
   "Return A-FILE-NAME but converted in to a spec file name."
   (concat
    (file-name-directory a-file-name)
-   (replace-regexp-in-string "\\(\\.rb\\)?$" "_spec.rb" (file-name-nondirectory a-file-name))))
+   (replace-regexp-in-string "\\(\\.\\(rb\\|rake\\)\\)?$" "_spec.rb" (file-name-nondirectory a-file-name))))
```

Comment:
I recommend [Edebug](https://www.gnu.org/software/emacs/manual/html_node/elisp/Using-Edebug.html). But please let me know if you don't manage to solve this by yourself.